credential-free/spring-cloud-azure-starter-jdbc-mysql/spring-cloud-azure-mysql-sample/README.md

```diff
@@ -0,0 +1,224 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- azure-database-mysql
+name: Store and retrieve information in Azure Database for MySQL in Spring Boot Application
+description: This example demonstrates how to use passwordless connections to store and retrieve information in Azure Database for MySQL in a Spring Boot application.
+---
+
+# Store and retrieve information in Azure Database for MySQL in Spring Boot Application
+
+This code sample demonstrates how to use passwordless connections to store and retrieve information in Azure Database for MySQL using the Spring Cloud Azure MySQL starter.
```

Comment:
It's better to add a link to `passwordless connections` and `Azure Database for MySQL`.